codespell_lib/data/dictionary.txt

```diff
@@ -712,6 +717,7 @@ alreasy->already
 alreay->already
 alreayd->already
 alredy->already
+alright->all right
```

Comment:
Not a typo https://www.merriam-webster.com/dictionary/alright

Reply:
From the same page:
«Although the spelling alright is nearly as old as all right, some critics have insisted alright is all wrong.»
«It is less frequent than all right but remains common especially in informal writing.»

Anyway, not a problem. I'm going to remove it.